features/camel/src/main/feature/feature.xml

```diff
@@ -18,7 +18,7 @@
 
     <repository>mvn:org.apache.karaf.features/standard/${karaf.version}/xml/features</repository>
 
-    <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
+    <repository>mvn:org.apache.karaf.features/spring-legacy/${karaf.version}/xml/features</repository>
```

Comment:
Curious what it means that it is `legacy` now. Do we need to switch up something to be using a more current spring feature?

Reply:
This has been removed. It was due to our version of camel being old, but now that it's been updated in the baseline this is no longer needed.